### PR TITLE
fix rootfs-propagation

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: completions/bash/oci-runtime-tool [hunk 1]
```diff
@@ -130,11 +130,9 @@ __oci-runtime-tool_complete_compliance_level() {
 __oci-runtime-tool_complete_propagations() {
 	COMPREPLY=( $( compgen -W "
 		private
-		rprivate
-		rshared
-		rslave
```
Comment (wking) [inline completions/bash/oci-runtime-tool]: This catches up with the 1.0 spec, but we may want to see how opencontainers/runtime-spec#923 plays out before removing them.  Or we can remove them now and restore them later if the the `r*` forms get added to the spec.

Comment (q384566678) [reply completions/bash/oci-runtime-tool]: I think we can remove them now, and make changes when spec changes.

**Comment by Mashimiao** [comment]
LGTM

**Comment by q384566678** [comment]
I have modified a similar situation in `generate.go`. PTAL

**Comment by q384566678** [comment]
ping @Mashimiao @liangchenye @hqhq @mrunalp

**Comment by Mashimiao** [comment]
LGTM

**Comment by q384566678** [comment]
@liangchenye @hqhq PTAL

File: completions/bash/oci-runtime-tool (hunk 1 above)
Comment (liangchenye) [reply completions/bash/oci-runtime-tool]: +1, we can remove them now.

**Comment by liangchenye** [comment]
LGTM

**Pull request merged**